FAQ-Plans (#22)
SolaceDev/maas-docs-online: docs/group_faqs/gf_faqs.rst

.. image:: ../img/faqs_enterprise_access_1.png
   :target: https://console.solace.cloud/services/create

> Q: What are plans in Solace PubSub+ cloud?

Reviewer: ```suggestion
Q: What are plans in Solace PubSub+ cloud?
```
```suggestion
Q: What are plans in PubSub+ cloud?
```

> A: On the Solace Cloud you could choose a plan to use for your messaging needs, below are the enterprise plans Solace PubSub+ cloud offer:

Reviewer: ```suggestion
A: On PubSub+ Cloud you could choose a plan to use for your messaging needs, below are the enterprise plans PubSub+ Cloud offers:
```

- Kilo
- Mega
- Giga
- Tera

> 

Reviewer: What about Developers, could we talk about the nano plan for developers?

Q: How are plans used?

A: There are two ways you could use an enterprise plan, you can ask for a Prepaid Subscription or for On-Demand services.

Q: How do I know what plan is best for my needs?

> A: To understand the details of the plans offered by Solace PubSub+ cloud, please visit the pricing page on our site, the plans differ on number of connections

Reviewer: ```suggestion
A: To understand the details of the plans offered by Solace PubSub+ cloud, please visit the pricing page on our site, the plans differ on number of Connections
```

Throughput, Usage and Storage:
`Plans Pricing <https://cloud.solace.com/pricing/>`_ page

Q: Can I change my plans ?

A: You could raise a request to change the plan best suitable for your needs and our production engineering team will help you with it.

> Q: How do I add a plan?

Reviewer: I'm at my limit, how can I get a higher plan limit.
